src/resonanceReconstruction/rmatrix/legacy/SpinGroupBase/src/makeReactionIdentifiers.hpp

```diff
@@ -0,0 +1,24 @@
+static
+std::vector< ReactionID >
+makeReactionIdentifiers( const Channel< Neutron >& channel,
+                         const ResonanceTableType& table ) {
+
+  std::vector< ReactionID > reactions;
+
+  // add elastic and capture
+  auto incident = channel.particlePair().particle().particleID();
+  auto target = channel.particlePair().residual().particleID();
+  reactions.push_back( ReactionID{ incident, target, ReactionType( "elastic" ) } );
+  reactions.push_back( ReactionID{ incident, target, ReactionType( "capture" ) } );
+
+  // add fission if it exists
+  if ( ranges::count_if(
+           table.resonances(),
+           [&] ( const auto& resonance )
+               { return resonance.hasFission(); } ) > 1 ) {
```

Comment:
I don't think it really matters, but wouldn't just looping over resonances and breaking if `true` be a lot more straightforward than ranges with a lambda?

Reply:
True. But again, it catches intent better. I could have declared the lambda in advance and then write:
```
if ( ranges::cound_if( table.resonances(), hasFission ) > 1 )
```
It's same intent as any:
```
if any( [ resonance.hasFission() for resonance in table.resonances() ] )
```